eoscostarica/guias.eoscostarica.io
# docs(frontend): Improve Readme

### GH Issue

Resolves #217  

### Steps to test
1. Go to Readme  file
2. Check content is right


#### CheckList
- [ ] The content is equal to https://github.com/eoscostarica/guide.eoscostarica.io
